.github/workflows/github-actions.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       # If you want to matrix build , you can append the following list.
       matrix:
         go_version:
-          - 1.13
+          - 1.15
```

Comment:
why update the go version?

Reply:
Because this pr contains package that using golang v1.5 feature. if not, when processing ci check, it may failed.